plotly_resampler/aggregation/aggregators.py

```diff
@@ -198,38 +202,9 @@ def _arg_downsample(
         n_out: int,
         **kwargs,
     ) -> np.ndarray:
-        # The block size 2x the bin size we also perform the ceil-operation
-        # to ensure that the block_size * n_out / 2 < len(x)
-        block_size = math.ceil(y.shape[0] / n_out * 2)
-
-        # Calculate the offset range which will be added to the argmin and argmax pos
-        offset = np.arange(0, stop=y.shape[0] - block_size, step=block_size)
-
-        # Calculate the argmin & argmax on the reshaped view of `s` &
-        # add the corresponding offset
-        argmin = (
-            y[: block_size * offset.shape[0]].reshape(-1, block_size).argmin(axis=1)
-            + offset
-        )
-        argmax = (
-            y[: block_size * offset.shape[0]].reshape(-1, block_size).argmax(axis=1)
-            + offset
-        )
-
-        # Note: the implementation below flips the array to search from
-        # right-to left (as min or max will always use the first same minimum item,
-        # i.e. the most left item)
-        # This however creates a large computational overhead -> we do not use this
-        # implementation and suggest using the minmaxaggregator.
-        # argmax = (
-        #     (block_size - 1)
-        #     - np.fliplr(
-        #         s[: block_size * offset.shape[0]].values.reshape(-1, block_size)
-        #     ).argmax(axis=1)
-        # ) + offset
-
-        # Sort the argmin & argmax (where we append the first and last index item)
-        return np.unique(np.concatenate((argmin, argmax, [0, y.shape[0] - 1])))
+        if x is None:
+            return MinMaxDownsampler().downsample(y, n_out=n_out)
```

Comment:
This is not a MinMax Overlap Aggregator.

Reply:
Should be MinMax aggregator

Reply:
whoops. my bad.